src/nvim/syntax.c

```diff
@@ -5751,185 +5751,94 @@ static void syntime_report(void)
 *  Highlighting stuff		      *
 **************************************/
 
-/*
- * The default highlight groups.  These are compiled-in for fast startup and
- * they still work when the runtime files can't be found.
- * When making changes here, also change runtime/colors/default.vim!
- * The #ifdefs are needed to reduce the amount of static data.  Helps to make
- * the 16 bit DOS (museum) version compile.
- */
-# define CENT(a, b) b
-static char *(highlight_init_both[]) =
-{
-  CENT(
-      "ErrorMsg term=standout ctermbg=DarkRed ctermfg=White",
-      "ErrorMsg term=standout ctermbg=DarkRed ctermfg=White guibg=Red guifg=White"),
-  CENT("IncSearch term=reverse cterm=reverse",
-      "IncSearch term=reverse cterm=reverse gui=reverse"),
-  CENT("ModeMsg term=bold cterm=bold",
-      "ModeMsg term=bold cterm=bold gui=bold"),
-  CENT("NonText term=bold ctermfg=Blue",
-      "NonText term=bold ctermfg=Blue gui=bold guifg=Blue"),
-  CENT("StatusLine term=reverse,bold cterm=reverse,bold",
-      "StatusLine term=reverse,bold cterm=reverse,bold gui=reverse,bold"),
-  CENT("StatusLineNC term=reverse cterm=reverse",
-      "StatusLineNC term=reverse cterm=reverse gui=reverse"),
+// The default highlight groups.  These are compiled-in for fast startup and
+// they still work when the runtime files can't be found.
+//
+// When making changes here, also change runtime/colors/default.vim!
+
+static char *highlight_init_both[] =
+{
+  "Conceal      ctermbg=DarkGrey ctermfg=LightGrey guibg=DarkGrey guifg=LightGrey",
+  "DiffText     cterm=bold ctermbg=Red gui=bold guibg=Red",
+  "ErrorMsg     ctermbg=DarkRed ctermfg=White guibg=Red guifg=White",
+  "IncSearch    cterm=reverse gui=reverse",
+  "ModeMsg      cterm=bold gui=bold",
+  "NonText      ctermfg=Blue gui=bold guifg=Blue",
+  "PmenuSbar    ctermbg=Grey guibg=Grey",
+  "StatusLine   cterm=reverse,bold gui=reverse,bold",
+  "StatusLineNC cterm=reverse gui=reverse",
+  "TabLineFill  cterm=reverse gui=reverse",
+  "TabLineSel   cterm=bold gui=bold",
+  "TermCursor   cterm=reverse gui=reverse",
+  "VertSplit    cterm=reverse gui=reverse",
+  "WildMenu     ctermbg=Yellow ctermfg=Black guibg=Yellow guifg=Black",
```

Comment:
Looks like `Conceal` and `WildMenu` got moved from `highlight_init_[dark,light]` to `_both`, but they do have the same values so I guess that's not a problem. Please mention it in the commit message.